packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -680,7 +680,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
       this.props.onItemClick(ev, item);
     }
 
-    this.dismiss(ev, true);
+    !ev.defaultPrevented && this.dismiss(ev, true);
```

Comment:
Can we add an example of this somewhere on the demo site so people can see how to use this?

Reply:
Hope to get to this tomorrow.

Reply:
A great example from the linked issue is to click on a contextual menu and have it change to an input

Reply:
or having a "see more" button that you can click and see more links in menu

Reply:
![context](https://user-images.githubusercontent.com/1434956/32116021-d1c0b08e-bafd-11e7-9e90-406e16dfede3.gif)